.github/workflows/operate-a11y.yml

```diff
@@ -1,6 +1,15 @@
 name: Operate a11y tests
 on:
   push:
+    branches:
+      - 'main'
+      - 'stable/**'
```

Comment:
🔧 Maybe we should also `release/*` as we have for Zeebe https://github.com/camunda/zeebe/blob/main/.github/workflows/zeebe-ci.yml#L8

Reply:
Actually that is probably something I should clarify with the Operate team since not even all their workflows have `main` + `stable/*` yet, some only have `main` and I don't know whether that's on purpose.

Reply:
@pedesen is there a general rule for Operate CI workflows like `operate-a11y.yml`, `operate-backend-linting.yml`, `operate-e2e-tests.yml` etcpp that they should all trigger on:

- `main`
- `stable/**`
- `release/**`

?

Reply:
@cmur2 So currently there is no Operate on the stable branches on this repo (so `main` is sufficient). But for the future it makes sense to run all of the test workflows on stable branches too, because Operate will be present in `stable/8.5` and future stables.